index.html

```diff
@@ -438,6 +438,16 @@ <h2>
         />
       </a>
     </footer>
+    <div class="menu-bar">
+      <i class="fa fa-times"></i>
+      <div class="header-slide-links">
+        <li><a >Portifolio</a></li>
+        <li><a >About</a></li>
```

Comment:
- [ ] I suggest that you add the ```href``` attribute to the anchor tags so as to take the user to the appropriate part of the page as mentioned in the project requirements.

![image](https://user-images.githubusercontent.com/107795045/210779478-928ce673-123b-40cc-bc4d-b7b597db123f.png)